test/ecto/adapters/clickhouse/connection_test.exs

```diff
@@ -2065,6 +2102,10 @@ defmodule Ecto.Adapters.ClickHouse.ConnectionTest do
            ]
   end
 
+  # TODO?
+  @tag :skip
+  test "drop constraint on cluster"
```

Comment:
https://hexdocs.pm/ecto_sql/Ecto.Migration.Constraint.html doesn't have `:options` field. We can hack around and pass cluster in `:comment` though :)

Reply:
We can also open a PR into Ecto to add `:options`

Reply:
I'm relatively new to ClickHouse, so maybe I'm just not fully aware, but according to the docs, constraints are created and dropped using `ALTER TABLE` statements https://clickhouse.com/docs/en/sql-reference/statements/alter/constraint

Reply:
Yes, but that's not the issue. The issue is that we we can't pass any `:options` to https://hexdocs.pm/ecto_sql/Ecto.Migration.html#constraint/3